app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ActivitiesController < ApplicationController
+	include ActivitiesHelper
+	before_filter :redirect_to_root_if_not_logged_in
+	before_filter :redirect_to_root_if_not_admin, only: :dashboard
+	before_action :set_todays_activity, except: :dashboard
+
+	def index
+		@activities = current_user.get_authorized_activities(user_id: params['user'])
```

Comment:
no, this user is different
the one which is selected

On Wed, May 20, 2015 at 4:14 PM, B V Satyaram <notifications@github.com>
wrote:

> In app/controllers/activities_controller.rb
> <https://github.com/chi6rag/HourGlass/pull/1#discussion_r30689243>:
>
> > @@ -0,0 +1,36 @@
> > +class ActivitiesController < ApplicationController
> > +	include ActivitiesHelper
> > +	before_filter :redirect_to_root_if_not_logged_in
> > +	before_filter :redirect_to_root_if_not_admin, only: :dashboard
> > +	before_action :set_todays_activity, except: :dashboard
> > +
> > +	def index
> > +		@activities = current_user.get_authorized_activities(user_id: params['user'])
>
> Something is wrong. You are calling on a user object and also passing a
> user.
> Only one should happen.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chi6rag/HourGlass/pull/1/files#r30689243>.
>